README.md

```diff
@@ -1,5 +1,25 @@
 # FMS HF Tuning
 
+- [Installation](#installation)
```

Comment:
@HarikrishnanBalagopal why is this section having so many lines of change. Only addition is line 20, isnt it?